sdk/python/foundation-models/benchmarking/evaluating_claude_models.ipynb

```diff
@@ -0,0 +1,205 @@
+    "\n",
+    "## Prerequistes\n",
+    "- An Azure account with an active subscription - [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)\n",
+    "- An Azure ML workspace with compute cluster - [Configure workspace](../../configuration.ipynb)\n",
+    "- Installed Azure Machine Learning Python SDK v2 - [install instructions](../../../README.md) - check the getting started section\n",
+    "- A python environment with [mlflow](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-mlflow-configure-tracking?view=azureml-api-2&tabs=python%2Cmlflow) for retrieving eval metrics\n",
+    "- Access keys for Claude endpoints on [Amazon Web Services Bedrock platform](https://aws.amazon.com/bedrock/claude/)  \n",
+    "\n",
+    "## Configuring a Workspace connection for Bedrock access\n",
+    "You will use a Workspace connection to securely store Bedrock access keys. Follow the steps below to create a custom-type connection:\n",
+    "- Follow directions for [creating a custom connection in the AzureML studio UI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/tools-reference/python-tool?view=azureml-api-2#create-a-custom-connection)\n",
+    "- Add the following two key-value pairs to the custom connection:\n",
+    "  1. A key named `AccessKey` with a value containing your AWS access key\n",
+    "  2. A key named `SecretKey` with a value containing your AWS secret access key \n",
+    "\n",
+    "## Configuring and running an evaluation pipeline\n",
+    "Please set global values in the following cell for your AzureML Workspace, the Bedrock endpoint you want to call, the name of connection you created in the previous step, and the name of the eval you want to run.\n",
+    "\n",
+    "Supported evals are the following: `boolq`, `gsm8k`, `hellaswag`, `human_eval`, `mmlu_humanities`, `mmlu_other`, `mmlu_social_sciences`, `mmlu_stem`, `openbookqa`, `piqa`, `social_iqa`, `winogrande`.\n",
+    "\n",
+    "You can also set the sample ratio, the fraction of the selected dataset to run for the eval.\n",
+    "\n",
+    "**Warning**: Many datasets contain thousands of examples which can lead to high endpoint usage costs. We advise starting with a small sample ratio (e.g., 1%) to verify the pipeline and then increasing the ratio if desired. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ce3f8a4-ce03-462b-89fa-19cac827c30c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# AzureML settings - please fill in your values\n",
+    "subscription_id = \"<Azure subscription ID>\"\n",
+    "resource_group = \"<Resource group>\"\n",
+    "workspace_name = \"<Workspace name\"\n",
+    "experiment_name = \"<Experiment name>\"\n",
+    "compute_name = \"<AzureML Compute name>\"\n",
```

Comment:
1. Can we specify user just needs CPU compute, maybe suggest a VM_SIZE that works?
2. Can we just create the compute for the user just like how we used to handle in automl notebooks? New users will have to learn how to create the compute.

Reply:
In the spirit of this comment, I've changed the pipelines/notebook to default to serverless compute so that the user need not know anything or setup anything related to compute clusters. I've added a link to docs on creating a cluster and some optional code to use one's own cluster